src/runtime/inline-cta-api-test.js

```diff
@@ -174,6 +174,25 @@ describes.realWin('InlineCtaApi', (env) => {
       expect(url).to.equal(resultUrl);
     });
 
+    it('getUrl returns url with language setting', () => {
+      clientConfigManagerMock
+        .expects('shouldForceLangInIframes')
+        .returns(true)
+        .once();
+      clientConfigManagerMock.expects('getLanguage').returns('pt-BR').once();
+      const urlPrefix = '/url_prefix';
+      const resultUrl =
+        'https://news.google.com/swg/ui/v1/url_prefix?_=_&origin=about%3Asrcdoc&configurationId=survey_config_id&isClosable=true&calledManually=false&previewEnabled=false&publicationId=pub1&ctaMode=CTA_MODE_INLINE&hl=pt-BR';
+
+      const url = inlineCtaApi.getUrl_(
```

Comment:
It doesn't seem ideal to be testing private methods of the class. Wondering if other tests could incorporate these asserts.

Reply:
Done